src/router.dispatch.ts

```diff
@@ -1,16 +1,19 @@
 // Do Router dispatch here, i.e. map incoming routes to appropriate router
-import { Application, Request, Response } from "express";
-import indexRouter from "./routers/indexRouter";
-import userRouter from "./routers/userRouter";
-import companyRouter from "./routers/companyRouter";
+import { Application, Router } from "express";
+import { HomeRouter, HealthCheckRouter } from "./routers";
+import { healthcheckUrl, servicePathPrefix } from "./utils/constants/urls";
+import { errorHandler, pageNotFound } from "./routers/handlers/errors";
 
 const routerDispatch = (app: Application) => {
-    app.use("/", indexRouter);
-    app.use("/user", userRouter);
-    app.use("/company", companyRouter);
-    app.use("*", (req: Request, res: Response) => {
-        res.status(404).render("partials/error_400");
-    });
+
+    const router = Router();
+    app.use(servicePathPrefix, router);
+
+    router.use("/", HomeRouter);
+    router.use(healthcheckUrl, HealthCheckRouter);
+
```

Comment:
We should add the session and auth middleware now. It will save us many headaches down the road.
Make sure they are attched to the router after the Home, and healthcheck endpoints so it won't be active on those pages. 
```suggestion
    router.use('/', sessionMiddleware)
    router.use('/', authMiddleware)
```